fedbiomed/node/cli.py

```diff
@@ -605,19 +612,31 @@ def initialize(self):
         class ConfigNameActionNode(ConfigNameAction):
 
             _this = self
-            _component = ComponentType.NODE
+            _component = ComponentType(2)
+
+            def set_component(self, config_name: str) -> None:
+                """Create node instance"""
+                print("NAME:", config_name)
+                config = NodeConfig(name=config_name)
+                node = Node(config)
+
+                # Set node object to make it accessible
+                setattr(ConfigNameActionNode._this, '_node', node)
+                os.environ[f"FEDBIOMED_ACTIVE_{self._component.name}_ID"] = \
+                    config.get("default", "id")
 
-            def import_environ(self) -> 'fedbiomed.node.environ.Environ':
-                """Imports dynamically node environ object"""
-                return importlib.import_module("fedbiomed.node.environ").environ
+                # Set node in all subparsers
+                for _, parser in ConfigNameActionNode._this._arg_parsers.items():
+                    setattr(parser, '_node', node)
 
         self._parser.add_argument(
             "--config",
             "-cf",
             nargs="?",
             action=ConfigNameActionNode,
             default="config_node.ini",
-            help="Name of the config file that the CLI will be activated for. Default is 'config_node.ini'.")
+            help="Name of the config file that the CLI will be activated for."
+                "Default is 'config_node.ini'.")
```

Comment:
Shouldn't we remove the last lines of the file ?

Same applies for `fedbiomed.researcher.cli`

```
if __name__ == '__main__':
    cli = NodeCLI()
    cli.parse_args()
```

This seems to be called now only from `fedbiomed_run` in `forward_command_to_python_cli` - which is not used anymore.

Reply:
Hi @mvesin,

Thanks. Yes we can remove the last lines

Reply:
And probably the same for `fedbiomed.common.cli` also

Reply:
pushed modif for that